.github/workflows/nix.yml

```diff
@@ -40,7 +40,7 @@ jobs:
     - name: Build and cache artifacts
       run: |
         echo Building the project and its devShell
-        nix build .#check --log-lines 500 --show-trace
+        nix build .?submodules=1#check --log-lines 500 --show-trace
```

Comment:
Note, this is required for CI to fetch the new marmalade tests (which are supplied via git submodules).